Mobile onboarding/connect-wallet screens

Pandelissym opened the pull request:
## Description

Adjusted the onboarding screen to be mobile friendly.
Adjusted the connect-wallet page to be mobile friendly including error toasts and kado pop up.
Adjusted the privacy screen to be mobile friendly.

Closes #110 

## Screenshots (if applicable)

https://github.com/Kryha/KREAd/assets/54764628/f7f28943-f611-4eb1-b6c6-f33e0f33da1b

## Checklist

Make sure all items are checked before submitting the pull request. Remove any items that are not applicable.

- [x] The PR title is clear and concise.
- [x] Are there changes in the /fronted folder? Make sure `cd frontend && yarn build` runs successfully.;

carlos-kryha commented:
the first paragraph in the "Who we are" section has some indentation issues
![Screenshot 2023-12-12 at 12 51 09](https://github.com/Kryha/KREAd/assets/97608823/f99f49e1-106d-40fe-a78d-42df71fb6615)

carlos-kryha commented:
would be good to add some top margin/padding to the `Who we are title`

Pandelissym commented:
@carlos-kryha I fixed the identation issue by removing the newline characters from the text, now they can take the whole space freely. Didnt seem to affect the desktop version in a bad way.

Also added padding to the "Who we are section". Let me know if it needs some more adjusting.

carlos-kryha commented:
> @carlos-kryha I fixed the identation issue by removing the newline characters from the text, now they can take the whole space freely. Didnt seem to affect the desktop version in a bad way.
> 
> Also added padding to the "Who we are section". Let me know if it needs some more adjusting.

oh right, I didn't realize we used newlines in the text consts that's kind of bad. Looking good now tho
